src/embeddedExplorer/examples/sandboxLocalDev.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>Embedded Explorer Example</title>
+  </head>
+  <body
+    style="
+      margin: 0;
+      overflow-x: hidden;
+      overflow-y: hidden;
+      height: 100vh;
+      width: 100vw;
+    "
+  >
+    <script>
+      {
+        // Note that the target div must be in the dom before the script tag creating the embedded explorer
+      }
+    </script>
+    <div
+      style="width: 800px; height: 700px; margin: auto; border: none"
+      id="embeddableSandbox"
+    ></div>
+    <noscript>You need to enable JavaScript to run this app.</noscript>
+    <script src="../../dist/embeddable-explorer.umd.production.min.js"></script>
```

Comment:
I'm running into `Uncaught TypeError: window.EmbeddedSandbox is not a constructor` error when I open sandboxLocalDev. Should this be `../../../dist/embeddable-sandbox.umd.production.min.js`?

Reply:
this whole file was actually an old development iteration, I'll delete it!